packages/clerk-js/src/core/resources/SignIn.ts

```diff
@@ -470,3 +490,132 @@ export class SignIn extends BaseResource implements SignInResource {
     };
   }
 }
+
+class SignInFuture implements SignInFutureResource {
```

Comment:
We can add JSDocs now or later to the methods here, but we should do it eventually! Saves us time later too 😄